requirements.txt

```diff
@@ -1,12 +1,13 @@
+aws-sam-cli
```

Comment:
Since this is invoked with subprocess, it doesn't make sense to pin this.

Reply:
Since we're invoking sam as a subprocess, it's not required so long as SAM is accessible on the PATH. I can remove this. I'll add sam install via pypi as an "extra" that can be installed this way.